sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **Breaking Changes**
 
 * `ServiceBusSender` and `ServiceBusReceiver` are no more reusable and will raise `ValueError` when trying to operate on a closed handler.
+* `ServiceBusMessage` will now raise a `TypeError` when provided an invalid body type.  Valid bodies are strings, bytes, None, or a list of string or bytes items.
```

Comment:
I'd rather not mention we support "a list of string or bytes items", because our current uamqp implementation would concatenate them into a single string which I think is a bit unexpected behavior.

(Besides we had discussion on supporting AmqpValue before, in which case a list of string/bytes should be a list not concatenation.)